cumulus_etl/etl/tasks/base.py

```diff
@@ -272,6 +272,7 @@ def _update_completion_table(self) -> None:
                     "export_time": self.task_config.export_datetime.isoformat(),
                     "export_url": self.task_config.export_url,
                     "etl_version": cumulus_etl.__version__,
+                    "etl_time": self.task_config.timestamp.isoformat(),
```

Comment:
What time to use here was a small choice. I initially had "whatever time it is right now as we're writing this completion entry" - but then I switched it to the global ETL timestamp, which gets set early in the ETL run. This means that the timestamp is a little less accurate to the exact time the entries get uploaded to Athena, but... we gain the ability to check which resources have the same timestamp (i.e. were in the same run) and we gain the ability to go look up the log files in S3 because they use the same global timestamp in their filename.